packages/scandipwa/src/store/Config/Config.action.ts

```diff
@@ -9,32 +9,16 @@
  * @link https://github.com/scandipwa/scandipwa
  */
 
-import { Device } from 'Type/Device.type';
-
 import {
     ConfigActionType,
     ConfigStore,
-    UpdateConfigAction,
-    UpdateConfigCurrencyAction,
-    UpdateConfigDeviceAction,
+    UpdateConfigStateAction,
 } from './Config.type';
 
-/** @namespace Store/Config/Action/updateCurrentCurrency  */
-export const updateCurrentCurrency = (selectedCurrency: string): UpdateConfigCurrencyAction => ({
-    type: ConfigActionType.UPDATE_CURRENT_CURRENCY,
-    selectedCurrency,
-});
-
-export const UPDATE_CONFIG = 'UPDATE_CONFIG';
-
-/** @namespace Store/Config/Action/updateConfig */
-export const updateConfig = (config: Partial<ConfigStore>): UpdateConfigAction => ({
-    type: ConfigActionType.UPDATE_CONFIG,
-    config,
-});
+export const UPDATE_CONFIG_STATE = 'UPDATE_CONFIG_STATE';
 
-/** @namespace Store/Config/Action/updateConfigDevice */
-export const updateConfigDevice = (device: Device): UpdateConfigDeviceAction => ({
-    type: ConfigActionType.UPDATE_CONFIG_DEVICE,
-    device,
+/** @namespace Store/Config/Action/updateConfigState */
+export const updateConfigState = (state: Partial<ConfigStore>): UpdateConfigStateAction => ({
+    type: ConfigActionType.UPDATE_CONFIG_STATE,
```

Comment:
Lets stay consist in naming of actions for updating. All others are updateSomething**Store**